### ISSUE TITLE
dayClick not working on Firefox mobile because of Firefox bug

### ISSUE THREAD

**Issue by Rouche** [fullcalendar/fullcalendar#3450]
### Bug Reports
I got a nosebleed on this one ><

Firefox mobile Touch even properties pageX / pageY are screwed. They have invalid values. (Seems to substract clientX / clientY from the touches pageX / pageY)

Only on mobile, only on Firefox.
Latest version used.

I was unable to debug remotelly so i used Firefox OS SImulator to discover the issue.

![Imgur](http://i.imgur.com/Z4Byejg.jpg)

### Feature Requests

in utils.js, would it be regression free to inverse the priority of `getEvX / getEvY` like this:

Tests seems ok:
```
PhantomJS 2.1.1 (Windows 8 0.0.0): Executed 2148 of 2148 SUCCESS (2 mins 47.126 secs / 2 mins 46.995 secs)
[13:47:23] Finished 'test:single' after 2.95 min
```
```
function getEvX(ev) {
    var touches = ev.originalEvent.touches;
    if (touches && touches.length > 0) {
        return touches[0].pageX;
    }

	if (ev.pageX !== undefined) {
		return ev.pageX;
	}
}


function getEvY(ev) {
    var touches = ev.originalEvent.touches;
    if (touches && touches.length > 0) {
        return touches[0].pageY;
    }

	if (ev.pageY !== undefined) {
		return ev.pageY;
	}
}
```

**Comment by arshaw** [comment]
thanks. would you mind throwing up a quick PR?

**Comment by nmorozov** [comment]
This solution has fixed my problem with dayClick on Cordova app.

**Comment by arshaw** [comment]
Released in [v3.2.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.2.0)

**Comment by hongquan** [comment]
Touch not working again on Firefox mobile, with FullCalendar v4 and v5.6.
